README.md

```diff
@@ -30,6 +30,7 @@ JOBS_KIO_URL            | yes        |                         | URL to [Kio](ht
 JOBS_SERVICE_USER_URL   | yes        |                         | URL to Service User API
 JOBS_MINT_STORAGE_URL   | yes        |                         | URL to Mint storage
 JOBS_ESSENTIALS_URL     | yes        |                         | URL of [essentials](https://github.com/zalando-stups/essentials). Used to verify scopes.
+JOBS_MINT_COWORKER_URL  | no         |                         | URL of [mint-coworker proxy](https://github.bus.zalan.do/teapot/mint-coworker). Used for proxying bucket writes to GCS.
```

Comment:
@harti2006 added the swagger spec for [mint-coworker](https://github.bus.zalan.do/teapot/mint-coworker), and removed the link to GHE.